scRNA-seq/README.md

```diff
@@ -7,9 +7,13 @@ It covers normalization and dimension reduction methods that can be used for bot
 
 The notebooks that comprise this module are (estimated time to complete notebook):
```

Comment:
```suggestion
The notebooks that comprise this module are:
```

We should probably get rid of this with the remove of the time estimates, but otherwise this LGTM after the other comment on this PR is addressed!